tests/integration/runner

```diff
@@ -415,8 +406,9 @@ if __name__ == "__main__":
         env_cleanup=env_cleanup,
         parallel=parallel_args,
         rand=rand_args,
-        opts=" ".join(args.pytest_args).replace("'", "\\'"),
-        tests_list=" ".join(args.tests_list),
+        opts=" ".join(
+            map(lambda x: shlex.quote(x).replace("'", "\\'"), args.pytest_args)
```

Comment:
I have a feeling that there should be easier way than `.quote(x).replace("'", "\\'")`, but who cares.

Reply:
There are more problems with this, but anyway after other fixes it can be converted to a plain `shlex.quote`.